docs/build/ios.md

```diff
@@ -42,6 +42,7 @@ Follow the instructions below to build ONNX Runtime for iOS.
 * Xcode, https://developer.apple.com/xcode/
 * CMake, https://cmake.org/download/
 * Python 3, https://www.python.org/downloads/mac-osx/
+* Nuget https://formulae.brew.sh/formula/nuget (used to download the protobuf compiler, which is needed to build ONNX Runtime)
```

Comment:
should we link to the MS instructions instead?
https://learn.microsoft.com/en-us/nuget/install-nuget-client-tools#macoslinux

Reply:
I looked at those but found them a little confusing. I thought a single step would be better. Let me know your thoughts

Reply:
suggested that because it seems like a more official source.